src/main/db/databaseStart.js

```diff
@@ -0,0 +1,11 @@
+import admin from 'firebase-admin';
```

Comment:
This is unnecesary, i'm doing that right now in my PR, so it could break the build, please review my PR and make the changes to use that